package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "eventsexpress_node_js",
+  "description": "EventsExpress project Node.js part",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "start": "cross-env NODE_ENV=production node ./server.js",
```

Comment:
Should be `node src/index.js`. Same for sta`rt:dev`

Reply:
I can`t, because in server app try connect to DB and if will be all right with connect app will do `app.listen`

Reply:
/+/